src/features/resources/ResourcesList.tsx

```diff
@@ -43,9 +62,18 @@ const ResourceName = styled.div`
 const ResourceImg = styled.img`
   width: 3rem;
 `;
-
-const ResourceIcon = styled(Icon)`
-  height: auto;
-`;
-
+const IconLookupCustom = iconName => {
```

Comment:
Can we move `logoMapping` and `IconLookupCustom` to a central place and make the two components import and use the same code? Maybe like `src/features/resources/resources-utils.tsx` or something along those things?